README.md

```diff
@@ -22,13 +22,20 @@ Additionally, this means that Nix can do a better job of enforcing purity where
 
 ## How?
 
-To install sbtix clone the sbtix git repo and run the following:
+To install sbtix either:
+
+```
+nix shell github:natural-transformation/sbtix
+```
+
+Or clone the sbtix git repo and:
+
 ```
-cd Sbtix
+cd sbtix
 nix-env -f . -i sbtix
 ```
 
-Sbtix provides a script which will connect your project to the sbtix global plugin and launch sbt, it does this by setting the `sbt.global.base` directory to `$HOME/.sbtix`.
+sbtix provides a number of scripts to generate and update Nix expressions to fetch your dependencies and build your sbt project. These scripts work by opening your project in sbt and loading an additional sbtix plugin (via the `sbt.global.base` directory to `$HOME/.sbtix`). After generation, you don't need the sbtix command-line tools to actually build your project from nix.
```

Comment:
> After generation, you don't need the sbtix command-line tools to actually build your project from nix.

This is actually important. Is it right to say sbtix is a developer tool and not part of CI tooling?

Reply:
This is true for the sbtix sbt plugin and command-line tools - the generated default.nix will still use some sbtix code (`sbtix.buildSbtProgram`) but does not depend on the sbt plugin or command-line tools AFAICT

Reply:
You are right. Generated nix files by sbtix should be enough to package in nix. The only thing I'm not so sure is whether the user still needs to write some glue code or not. I will try to find out.

Reply:
I just tried it for https://codeberg.org/eicas/server-config/src/branch/main/calendar-export , the glue code is either generated or can be copied from the README. We can probably polish this further (the generated code and the README don't currently match exactly) but it seems to work pretty well already.

Reply:
You are right. I should revisit the glue code in my own project and see what we miss in the README.